src/storage/btree.h

```diff
@@ -351,11 +351,22 @@ struct btree_checkscan
   BTREE_ISCAN_OID_LIST oid_list;	/* Data area to store OIDs */
 };				/* B+tree <key-oid> check scan structure */
 
+struct btree_ovfl_oid_capacity
+{
+  int max_pg_cnt_per_key;	/* Distinct key count with overflow oid page) */
+  int dis_key_cnt;		/* Distinct key count (with ovfl pages) */
```

Comment:
An overflow page occurs when there are too many oids with the same key to store in one leaf node.

In overflow page, oids have the same key value. What does distinct key mean?

Reply:
Duplicate keys do not always have overflow oid pages.
This variable counts the number of distinct keys with overflow oid pages.